README.md

```diff
@@ -1,32 +1,73 @@
 
-## How to contribute with (or to) a deployment scenario?
-You are warmly invited to contribute to the IDS Deployment Scenarios in two ways: 
+Sharing deployment scenarios with others can also have many advantages for the original implementor, including:
+
+- **Getting recognized**: Sharing deployment scenarios with others can help the original implementor get recognized for their work. By sharing their scenarios, they can showcase their expertise and demonstrate their ability to solve complex problems.
+
+- **Feedback and improvement**: When others use the deployment scenarios, they may provide feedback and suggest improvements. This can help the original implementor refine their work and make it even better.
+
+- **Collaboration**: Sharing deployment scenarios with others can foster collaboration and create a community of people who are interested in similar topics. This can lead to new partnerships, joint projects, and opportunities to learn from others.
 
-:triangular_flag_on_post: Either by suggesting to list a deployment scenario here by using [the Data Space Radar form](https://forms.office.com/Pages/ResponsePage.aspx?id=NNZGs_usx0K9RPFVfuibG3WVHeFvj2hHgjU7ZCgshUhUMExMOTdCWDNMSERJTjlIUlRKMVc0QTUxMCQlQCN0PWcu). 
+- **Feel good factor**: Sharing deployment scenarios with others can give the original implementor a sense of satisfaction and fulfillment. Knowing that their work is helping others and making a positive impact can be very rewarding.
 
-:bulb: Or by expressing your idea, suggest something new (or asking a question) on a specific deployment scenario by creating an issue, by checking the contribution rules on [CONTRIBUTING.md](CONTRIBUTING.md), and the [Code of Conduct](CODE_OF_CONDUCT.md).
+### Relationship with Data Space Radar
+In the [Data Space Radar](https://internationaldataspaces.org/adopt/data-space-radar/), all solutions that reached the pilot and live stages can be considered as an IDS Deployment Scenario, since they involve the implementation of a data space solution to solve a specific business problem. 
+
+The deployment scenarios listed here are not necessarily attached to a particular use case in the Dataspace Radar. The relationship of IDS Deployment Scenarios with other assets of IDSA is explained in more detail in the [Deployment Scenarios Report](/DEPLOYMENT-SCENARIOS-REPORT.MD#23-ids-deployment-scenarios).
+
+### How to assess the maturity level of a IDS Deployment Scenario?
+The maturity level of an IDS Deployment Scenario is currently not listed but as more IDS deployment scenarios are listed, we will have this activated. 
+
+Meanwhile, please feel free to check and comment on the [Maturity Assessment Criteria](Maturity-Assessment-Criteria.md) on this repository that suggests a methodology to assess the maturity of IDS Deployment Scenarios, in three levels: Sandbox, Incubated and Graduated. 
+
+## How can I share a deployment scenario?
```

Comment:
please check the capitalization --> deployment scenario is in the text Deployment Scenario, please check in the text in general and use one version with capitals or without